tests/x11/keyboard_layout_gnome.pm

```diff
@@ -0,0 +1,79 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+
+# Summary: Test if gnome login manager honors keyboard layout changes
+# Maintainer: Dominik Heidler <dheidler@suse.de>
+
+use base "x11test";
+use strict;
+use testapi;
+use utils;
+use power_action_utils 'power_action';
+
+sub reboot {
```

Comment:
Somehow my comment was lost, do you think it makes sense to put it in `power_action_utils`. I guess we do same thing in some other tests as well.

Reply:
I would say better to think about refactoring power_action to wait_boot at the end because it happens very often. But also I think it is not the topic of this PR. Merging